src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Container, Image, VStack, Center } from '@chakra-ui/react';
+import './ErrorPage.css';
+import image2 from '../assets/images/image2.png';
+
+const ErrorPage = () => {
+  return (
```

Comment:
[nit] also if we have just the return let's just do without the `{}` brackets like

```JavaScript
const ErrorPage = () => (
    <Container ....>
        ...
    </Container>
);
```